Sources/Sentry/SentryReplayApi.m

```diff
@@ -22,18 +23,49 @@ - (void)unmaskView:(UIView *)view
 
 - (void)pause
 {
-    SentrySessionReplayIntegration *replayIntegration =
-        [SentrySDK.currentHub getInstalledIntegration:SentrySessionReplayIntegration.class];
+    SentrySessionReplayIntegration *replayIntegration
+        = (SentrySessionReplayIntegration *)[SentrySDK.currentHub
+            getInstalledIntegration:SentrySessionReplayIntegration.class];
     [replayIntegration pause];
 }
 
 - (void)resume
 {
-    SentrySessionReplayIntegration *replayIntegration =
-        [SentrySDK.currentHub getInstalledIntegration:SentrySessionReplayIntegration.class];
+    SentrySessionReplayIntegration *replayIntegration
+        = (SentrySessionReplayIntegration *)[SentrySDK.currentHub
+            getInstalledIntegration:SentrySessionReplayIntegration.class];
     [replayIntegration resume];
 }
 
+- (void)start
+{
+    SentrySessionReplayIntegration *replayIntegration
+        = (SentrySessionReplayIntegration *)[SentrySDK.currentHub
+            getInstalledIntegration:SentrySessionReplayIntegration.class];
+
+    if (replayIntegration == nil) {
+        replayIntegration = [[SentrySessionReplayIntegration alloc] init];
+
+        SentryOptions *options = [[SentryOptions alloc] init];
+        options.enableSwizzling = SentrySDK.currentHub.client.options.enableSwizzling;
+        options.experimental.sessionReplay.sessionSampleRate = 1;
+        __unused BOOL installed = [replayIntegration installWithOptions:options];
```

Comment:
`h`: This can break easily. What if parts of the replay integration rely on other options? I think we must not create a new instance of the options. Instead, we should use the existing options.

Reply:
A manually started replay integration should be controlled by its own set of options.
But there are options missing in here.

Maybe we could offer a parameter in the start function